common/src/main/java/io/seata/common/util/ReflectionUtil.java

```diff
@@ -41,40 +41,41 @@
      * @return the class by name
      * @throws ClassNotFoundException the class not found exception
      */
-    public static Class<?> getClassByName(String className) throws ClassNotFoundException{
+    public static Class<?> getClassByName(String className) throws ClassNotFoundException {
         return Class.forName(className, true, Thread.currentThread().getContextClassLoader());
     }
 
     /**
      * get Field Value
      *
-     * @param target the target
+     * @param target    the target
      * @param fieldName the field name
      * @return field value
      * @throws NoSuchFieldException the no such field exception
      * @throws SecurityException the security exception
      * @throws IllegalArgumentException the illegal argument exception
      * @throws IllegalAccessException the illegal access exception
      */
-    public static Object getFieldValue (Object target, String fieldName) throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
+    public static Object getFieldValue(Object target, String fieldName)
+        throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
```

Comment:
Why change the line here ？
I think we should have more characters on each line to avoid more line breaks.